ClientRuntimes/NodeJS/ms-rest/lib/requestPipeline.js

```diff
@@ -142,7 +142,7 @@ exports.requestLibrarySink = function (requestOptions) {
 exports.create = function (requestOptions) {
   return function () {
     if (arguments.length === 0) {
-      return exports.createWithSink(exports.requestLibrarySink);
+      return exports.createWithSink(exports.requestLibrarySink(requestOptions));
```

Comment:
How does this help?
If arguments.length === 0; which means nothing was passed as an argument while calling the create method. Then what would you gain by passing undefined (as requestOptions would be undefined in this scenario) to requestLibrarySink.

If something was passed as an argument while calling the create method then that something will be passed to  requestLibrarySink in the return statement on line 149.